help/dark-theme.md

```diff
@@ -7,15 +7,31 @@ for working in a dark space.
 
 {start_tabs}
 
-{settings_tab|display-settings}
+{tab|desktop-web}
 
-2. Under **Theme**, configure **Color scheme**.
+{settings_tab|display-settings}
 
-{end_tabs}
+1. Under **Theme**, configure **Color scheme**.
 
 The default is **Sync with computer**, which detects which theme to use based
 on the color scheme used by your operating system.
 
 You can also specify **Dark theme** or **Light theme** if you'd like
 Zulip to use the same color scheme regardless of your operating system
 configuration.
+
+{tab|mobile}
+
+{!mobile-profile-menu.md!}
+
+1. Tap **Settings**.
+
+1. Toggle **Night mode**.
+
+{end_tabs}
+
+## Related articles
+
+* [Configure default new user settings](/help/configure-default-new-user-settings)
+* [Create your organization profile](/help/create-your-organization-profile)
```

Comment:
Hm, I'm not sure the organization profile one fits here. How about this?

* [Set up your account](/help/set-up-your-account)
* [Review your settings](/help/review-your-settings)
* [Configure default new user settings](/help/configure-default-new-user-settings)